package.json

```diff
@@ -138,7 +139,7 @@
     "shortid": "^2.2.16",
     "simplebar-react": "^2.3.5",
     "slugify": "^1.6.0",
-    "stripe": "^8.170.0",
+    "stripe": "^9.8.0",
```

Comment:
https://github.com/stripe/stripe-node/wiki/Migration-Guide-for-v9

we don't use this API that was a breaking change so I moved us to latest here